src/NormalDayEncounter.java

```diff
@@ -19,9 +23,20 @@ public EncounterResult encounter(World world, Player playerInEncounter)
         }
         else
         {
-            player.tell("You rest all day and you feel great");
-            world.daysRemaining++;
-            result.playerIsAlive = true;
+            if (Dice.d20() < 18)
+            {
+                player.tell("You rest all day and you feel great");
+                world.daysRemaining++;
+                result.playerIsAlive = true;
+            }
+            else
+            {
+                player.tell("You found a bottle of medicine near your campsite!");
+                Item medicine = new Item(ItemType.MEDICINE, "A bottle of medicine");
```

Comment:
Describe the medicine better? Do we have multiple types? is it strong, medium weak?